src/components/ChatGhostUI/ContentLoadingWrapper/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ContentLoader from 'react-content-loader';
+
+const propTypes = {
+    /* Children to wrap within the Content Loader */
+    children: PropTypes.node.isRequired,
+
+    /* Height of the content-loader */
+    height: PropTypes.number.isRequired,
+};
+
+const ContentLoadingWrapper = (props) => {
+    <ContentLoader height={props.height} width="100%" title="">
```

Comment:
A comment here about why `title=""` is passed.

Reply:
**question (blocker):** Why is `title` passed with `""`?

Reply:
Default value for `title` is "Loading..." in `react-content-loader`. Hence I have to pass it empty to hide it.